Tasks/VsTest/vstest.ts

```diff
@@ -963,4 +939,59 @@ function isNullOrWhitespace(input) {
         return true;
     }
     return input.replace(/\s/g, '').length < 1;
-}
\ No newline at end of file
+}
+
+class VSTestVersion {
```

Comment:
>VSTestVersion [](start = 6, length = 13)

VsTestVersion should be an interface.

Typescript supports anonymous interface implementation. Dev14/Dev15 these shouldn't be classes. they should be anonymous implementations